### PR TITLE
Handle version javadoc tag conversion

### PR DESCRIPTION
**Pull request by akurtakov**
Fixes polluting the log with "Not supported yet conversion of DCVersion to element" and probably smth real too.

### CODE REVIEW

File: org.eclipse.jdt.core.javac/src/org/eclipse/jdt/core/dom/JavadocConverter.java [hunk 1]
```diff
@@ -135,6 +136,9 @@ private Optional<TagElement> convertBlockTag(DCTree javac) {
 		} else if (javac instanceof DCSince since) {
 			res.setTagName(TagElement.TAG_SINCE);
 			since.body.stream().map(this::convertElement).forEach(res.fragments::add);
+		} else if (javac instanceof DCVersion version) {
+            res.setTagName(TagElement.TAG_VERSION);
+            version.body.stream().map(this::convertElement).forEach(res.fragments::add);
```
Comment (fbricon) [inline org.eclipse.jdt.core.javac/src/org/eclipse/jdt/core/dom/JavadocConverter.java]: nit: can you fix the indentation?

Comment (akurtakov) [reply org.eclipse.jdt.core.javac/src/org/eclipse/jdt/core/dom/JavadocConverter.java]: Fixed, sorry about that.

**Comment by fbricon** [comment]
you need to use tabs
Screenshot 2024-05-22 at 16 28 45


you should always show whitespaces in your IDE ;-)

**Comment by akurtakov** [comment]
@fbricon In Eclipse I hit TAB once and got this 2 tabs+ 4 spaces automatically. 
What do you guys think about pushing settings (
image
) so we have unified settings?

**Pull request merged**